Patch to upgrade bluez-utils to version 5.0 and install gatttool.

diff -Naur a/package/bluez_utils/bluez_utils-install_gatttool.patch b/package/bluez_utils/bluez_utils-install_gatttool.patch
--- a/package/bluez_utils/bluez_utils-install_gatttool.patch	1970-01-01 01:00:00.000000000 +0100
+++ b/package/bluez_utils/bluez_utils-install_gatttool.patch	2013-11-13 02:35:14.147018259 +0000
@@ -0,0 +1,15 @@
+diff -Naur a/Makefile.tools b/Makefile.tools
+--- a/Makefile.tools	2013-10-15 14:41:07.000000000 +0100
++++ b/Makefile.tools	2013-11-13 02:19:52.334984046 +0000
+@@ -216,8 +216,9 @@
+ endif
+ 
+ if READLINE
+-noinst_PROGRAMS += attrib/gatttool \
+-			tools/obex-client-tool tools/obex-server-tool \
++bin_PROGRAMS    += attrib/gatttool
++
++noinst_PROGRAMS += tools/obex-client-tool tools/obex-server-tool \
+ 			tools/bluetooth-player tools/obexctl
+ 
+ attrib_gatttool_SOURCES = attrib/gatttool.c attrib/att.c attrib/gatt.c \
diff -Naur a/package/bluez_utils/bluez_utils.mk b/package/bluez_utils/bluez_utils.mk
--- a/package/bluez_utils/bluez_utils.mk	2013-11-13 02:50:56.621450242 +0000
+++ b/package/bluez_utils/bluez_utils.mk	2013-11-13 02:35:14.147018259 +0000
@@ -4,12 +4,12 @@
 #
 #############################################################
 
-BLUEZ_UTILS_VERSION = 4.101
+BLUEZ_UTILS_VERSION = 5.10
 BLUEZ_UTILS_SOURCE = bluez-$(BLUEZ_UTILS_VERSION).tar.gz
 BLUEZ_UTILS_SITE = $(BR2_KERNEL_MIRROR)/linux/bluetooth
 BLUEZ_UTILS_INSTALL_STAGING = YES
-BLUEZ_UTILS_DEPENDENCIES = dbus libglib2
-BLUEZ_UTILS_CONF_OPT = --enable-test --enable-tools
+BLUEZ_UTILS_DEPENDENCIES = dbus libglib2 readline libical udev
+BLUEZ_UTILS_CONF_OPT = --enable-test --enable-tools --disable-systemd
 BLUEZ_UTILS_AUTORECONF = YES
 BLUEZ_UTILS_LICENSE = GPLv2+ LGPLv2.1+
 BLUEZ_UTILS_LICENSE_FILES = COPYING COPYING.LIB
diff -Naur a/package/bluez_utils/Config.in b/package/bluez_utils/Config.in
--- a/package/bluez_utils/Config.in	2013-11-13 02:50:56.621450242 +0000
+++ b/package/bluez_utils/Config.in	2013-11-13 02:35:14.147018259 +0000
@@ -6,6 +6,9 @@
 	depends on BR2_USE_MMU # dbus
 	select BR2_PACKAGE_DBUS
 	select BR2_PACKAGE_LIBGLIB2
+	select BR2_PACKAGE_READLINE
+	select BR2_PACKAGE_LIBICAL
+	select BR2_PACKAGE_UDEV
 	help
 	  bluez utils
 
